app/src/main/AndroidManifest.xml

```diff
@@ -32,5 +32,11 @@
         </activity>
         <activity android:name=".screens.entrysubmission.entryimage.EntryImageActivity">
         </activity>
+        <activity
+            android:name=".screens.entrysubmission.join.JoinActivity"
```

Comment:
I think its likely we will move this package later, but for now lets keep it simple and leave it here.